### DISCUSSION TITLE
How can redirect to API Gateway done through AWS Lambda@Edge?

### DISCUSSION THREAD

**Discussion by meetbha** [serverless/serverless#11150] [Q&A]
I am getting below error when I do a redirect from Lambda@Edge to API Gateway.
Can some please help me?
	
<H1>403 ERROR</H1>
	<H2>The request could not be satisfied.</H2>
	<HR noshade size="1px">
	Bad request.
	We can't connect to the server for this app or website at this time. There might be too much traffic or a
	configuration error. Try again later, or contact the app or website owner.
	<BR clear="all">
If you provide content to customers through CloudFront, you can find steps to troubleshoot and help prevent this error by reviewing the CloudFront documentation.

**Comment by le2sky** [comment]
hello! @meetbha 
I don't know the exact solution cuz the situation is not detailed. 
if you are using other aws services together, check the region!
